frontend/src/components/CreateEmployee.jsx

```diff
@@ -1,24 +1,18 @@
+import React from "react";
 import { useState } from "react";
+import {useNavigate} from "react-router-dom"
 import "./createEmployee.css";
 
 const CreateEmployee = ({ employees, setEmployees, setIsCreateNew }) => {
   const [newFirstName, setNewFirstName] = useState("");
```

Comment:
No need of using 3 different state variables 
Combine these 3 into one state variable like this 
const [formData,setFormData] = useState({})

Reply:
I hate you sfm

Reply:
Resolve this asap

Reply:
I can't